onetimepass/otp.py

```diff
@@ -1,22 +1,27 @@
 import enum
+import functools
+import pathlib
 import random
-import secrets
 import string
 from typing import Dict
 
 import click
 
+from onetimepass import settings
+from onetimepass.db.json_encrypted import JSONEncryptedDB
 
-KEY = secrets.token_hex()
-
-
-def keyring_installed():
-    try:
-        import keyring
-    except ImportError:
-        return False
-    else:
-        return True
+try:
+    import keyring  # noqa 'keyring' in the try block with 'except ImportError' should also be defined in the except block
+except ImportError:
+    KEYRING_INSTALLED = False
+else:
+    KEYRING_INSTALLED = True
+    KEYRING_PARAMS = {
+        "service_name": "onetimepass",
```

Comment:
What do you think to move these params to settings.py file?

Reply:
👍